scss/_settings_animations.scss

```
@@ -12,8 +12,12 @@ $animation-easing: (
   in: cubic-bezier(0.55, 0.055, 0.675, 0.19),
 );
 
-@mixin vf-animation($property: all, $duration: brisk, $easing: out) {
+@mixin vf-transition($property: all, $duration: brisk, $easing: out) {
   transition-duration: map-get($animation-duration, $duration);
   transition-property: $property;
   transition-timing-function: map-get($animation-easing, $easing);
 }
+
+@mixin vf-animation($args...) {
```

Please add a comment about deprecation

```suggestion
// DEPRECATED: vf-animation mixin is deprecated and will be removed in future version of Vanilla, please use vf-transition instead
@mixin vf-animation($args...) {
```